tests/Doctrine/Website/Tests/Docs/resources/example-project/docs/en/index.rst

```diff
@@ -66,3 +66,37 @@ Reference Anchor
 - :ref:`Cross Ref Section A <cross_ref_section_a_anchor>`
 
 .. _`TestLink`: https://www.doctrine-project.org
+
+Glob TOC
+--------
+
+.. tocheader:: Glob TOC Title
+
+.. toc::
+
+.. toctree::
+    :depth: 3
+    :glob:
+
+    *
+
+TOC
+---
+
+.. tocheader:: TOC Title
+
+.. toc::
+
+.. toctree::
+    :depth: 3
+    :glob:
```

Comment:
@jwage this does not make any sense to me. Can you please explain why you use glob here when you build the TOC explicitly afterwards? Is it necessary?

Reply:
This is a test for the ToctreeDirective which supports glob wildcards for building table of contents. We still use this in other places in the docs and if a project does not provide a sidebar.rst it will fall back to generating one automatically.

Reply:
@jwage sure, but you have glob and explicit TOC declaration together in this example. The example before already uses glob. This is what confuses me here

Reply:
Oh sorry, I misunderstood. You are right, the glob option can be removed here since I do not pass any paths with a `*` in them.